packages/functional-tests/tests/syncV3/fxDesktopV3ForceAuth.spec.ts

```diff
@@ -151,8 +151,8 @@ test.describe('severity-1 #smoke', () => {
     }) => {
       const config = await configPage.getConfig();
       test.skip(
-        config.showReactApp.signUpRoutes === true,
-        'force_auth is no longer supported for signup with react'
+        config.showReactApp.resetPasswordRoutes === true,
+        'Scheduled for removal as part of React conversion (see FXA-8267).'
```

Comment:
Are you sure about changing this flag?

```suggestion
        config.showReactApp.signUpRoutes === true,
        'force_auth is no longer supported for signup with react'
```

Reply:
See above comment.